File: GestionIntApi/DTO/DetalleClienteDTO.cs
namespace GestionIntApi.DTO
{
    public class DetalleClienteDTO
    {
        public int Id { get; set; }

        public string NumeroCedula { get; set; }
        public string NombreApellidos { get; set; }
        public string Telefono { get; set; }
        public string Direccion { get; set; }

        public string FotoClienteUrl { get; set; }

        public string FotoContrato { get; set; }
        public string FotoCelularEntregadoUrl { get; set; }
    }
}

File: lib/models/detalle_cliente_dto.dart
class DetalleClienteDTO {
  int id;
  String numeroCedula;
  String nombreApellidos;
  String telefono;
  String direccion;
  String? fotoClienteUrl;
  String? fotoContrato;
  String? fotoCelularEntregadoUrl;

  DetalleClienteDTO({
    this.id = 0,
    required this.numeroCedula,
    required this.nombreApellidos,
    required this.telefono,
    required this.direccion,
    this.fotoClienteUrl,
    this.fotoContrato,
    this.fotoCelularEntregadoUrl,
  });

  Map<String, dynamic> toJson() => {
    'Id': id,
    'NumeroCedula': numeroCedula,
    'NombreApellidos': nombreApellidos,
    'Telefono': telefono,
    'Direccion': direccion,
    'FotoClienteUrl': fotoClienteUrl,
    'FotoContrato': fotoContrato,
    'FotoCelularEntregadoUrl': fotoCelularEntregadoUrl,
  };
}